src/api/app/controllers/webui/package_controller.rb

```diff
@@ -663,6 +663,16 @@ def remove
     end
   end
 
+  def trigger_services
+    begin
+      Suse::Backend.post "/source/#{URI.escape(@project.name)}/#{URI.escape(@package.name)}?cmd=runservice&user=#{User.current}"
+      flash[:notice] = 'Services successfully triggered'
+    rescue RuntimeError, Timeout::Error, ActiveXML::Transport::NotFoundError, ActiveXML::Transport::Error => e
```

Comment:
@Ana06 RuntimeError would be generic exceptions, right? That would mean we might miss possible bugs.